shenyu-web/src/main/java/org/apache/shenyu/web/configuration/ShenyuExtConfiguration.java

```diff
@@ -50,7 +51,6 @@ public ShenyuResult<?> shenyuResult() {
     @Bean
     @ConditionalOnMissingBean(value = RemoteAddressResolver.class, search = SearchStrategy.ALL)
     public RemoteAddressResolver remoteAddressResolver() {
-        return new RemoteAddressResolver() {
-        };
+        return new ForwardedRemoteAddressResolver(1);
```

Comment:
this code alreay in ShenyuConfiguration.java. 

can remove this ?

Reply:
yes，you can delete this code。

Reply:
you can remove this (modify this pr)